app/assets/stylesheets/blacklight/_pagination.scss

```diff
@@ -14,3 +14,20 @@
     flex-wrap: wrap;
   }
 }
+
+
+.link-pagination {
+  .disabled a {
```

Comment:
This rule should be unnecessary if you set `.page-item` on the node. https://getbootstrap.com/docs/5.3/components/pagination/#disabled-and-active-states